source/form.html

```diff
@@ -0,0 +1,143 @@
+              <input class="form-wrapper__check-box-input" type="checkbox" name="beach" checked>
+            </label>
+          </li>
+          <li>
+            <label class="form-wrapper__check-box-lebel">Видел Чака Норриса
+              <input class="form-wrapper__check-box-input" type="checkbox" name="norris">
+            </label>
+          </li>
+          <li>
+            <label class="form-wrapper__check-box-lebel">Накупил сувениров
+              <input class="form-wrapper__check-box-input" type="checkbox" name="souvenirs">
+            </label>
+          </li>
+          <li>
+            <label class="form-wrapper__check-box-lebel">Удержал башню
+              <input class="form-wrapper__check-box-input" type="checkbox" name="tower" checked>
+            </label>
+          </li>
+          <li>
+            <label class="form-wrapper__check-box-lebel">Разгромил отель
+              <input class="form-wrapper__check-box-input" type="checkbox" name="hotel">
+            </label>
+          </li>
+        </ul>
+      </div>
+      <div class="form-wrapper__block-personal-data">
+        <label class="form-wrapper__phone-number">номер телефона
+          <input class="form-wrapper__tel" type="tel" name="phone-number" placeholder="Номер, пожалуйста">
+        </label>
+        <label class="form-wrapper__mail-address">адрес почты
+          <input class="form-wrapper__email" type="email" name="mail-address" placeholder="Введите почту *" required>
+        </label>
+      </div>
+      <div class="form-wrapper__block-application-type">
+        <h2 class="form-wrapper__title-application-type">с каким приложением путешествовали?</h2>
+        <label class="form-wrapper__application-ios">Pink для iOS
+          <input class="form-wrapper__ios" type="radio" name="application-type" value="ios" checked>
+        </label>
+        <label class="form-wrapper__application-android">Pink на Android
+          <input class="form-wrapper__android" type="radio" name="application-type" value="android">
+        </label>
+        <label class="form-wrapper__application-windows-phone">Windows Phone
+          <input class="form-wrapper__windows-phone" type="radio" name="application-type" value="windows-phone">
+        </label>
+      </div>
+      <div class="form-wrapper__block-description-emotions">
+        <label class="form-wrapper__description-emotions">опишите свои эмоции
+          <textarea placeholder="Можно прям в красках, не стесняясь в выражениях"></textarea>
+        </label>
+      </div>
+      <button class="form-wrapper__push-contest-data" type="submit" name="button-to-send-contest-data" aria-label="кнопка для отправки данных конкурса.">отправить форму</button>
+      <p class="form-wrapper__required-fields"><span>*</span> — обязательные поля</p>
+    </form>
+  </main>
+
+  <footer class="footer-block">
+    <a сclass="link-footer__logo" href="#"><img class="link-footer__image" src="#" alt="Текстовый логотип Pink."></a>
```

Comment:
```suggestion
    <a class="link-footer__logo" href="#"><img class="link-footer__image" src="#" alt="Текстовый логотип Pink."></a>
```

поставь себе ВебШторм, он сразу такое подсвечивает